src/edu/psu/sweng500/team8/gui/GridPanel.java

```diff
@@ -73,6 +85,23 @@ public void populatePanel(CellGrid grid, GameSession gameSession) {
             }
         }
     }
+    
+    public void remarkGivenCells(CellGrid grid){
```

Comment:
I don’t understand why it should make a difference, but without it, when I save, the highlights disappear.